packages/camera/camera_android/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.10.8+9
+
+* Update annotations lib to 1.7.0.
```

Comment:
Here and in other places, I believe this should be "Updates..."

https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#changelog-style

Reply:
Yeah, thanks this always bites me. I need to write a pre commit script for this.